spec/services/username_changer_spec.rb

```diff
@@ -591,4 +603,70 @@ def username_and_something_else(username)
     end
   end
 
+  describe '#override' do
+    common_test_cases = [
+      [
+        "overrides the username if a new name is different",
+        "john", "bill", "bill", false
+      ],
+      [
+        "does not change the username if a new name is the same",
+        "john", "john", "john", false
+      ],
+      [
+        "overrides the username if a new name has different case",
+        "john", "JoHN", "JoHN", false
+      ]
+    ]
+
+    context "unicode_usernames is off" do
+      before do
+        SiteSetting.unicode_usernames = false
+      end
+
+      [
+        *common_test_cases,
+        [
+          "overrides the username if a new name after unicode normalization is different only in case",
```

Comment:
this test case description seems misleading given that `current` and `overrode` are both "john"

Reply:
@gschlager, I've renamed the test case, I probably copied it from another one and forgot to rename it.

Reply:
Oops, sorry, I haven't seen it's already approved. Thank you!